core/src/main/scala/com/pingcap/tispark/TiUtils.scala

```diff
@@ -80,7 +80,7 @@ object TiUtils {
         // bit/duration type is not allowed to be pushed down
         case attr: AttributeReference if nameTypeMap.contains(attr.name) =>
           val head = nameTypeMap.get(attr.name).head
-          return !head.isInstanceOf[BitType] && head.getType != MySQLType.TypeDuration
+          return !head.isInstanceOf[BitType]
```

Comment:
so it means that we can push down time now? is there any tests supporting it?

Reply:
Manually test.  We better refactor explain message in order to know whether a filter is pushed down or not.

Reply:
we have it in explain message already. You can observe the pushed-down filters with `Pushdown Filter: ` pattern.

Reply:
a filter will be either in `Pushdown Filter` or `Residual Filter`